### PR TITLE
Add docs about contributing to docs

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Offers guidance for community members who want to contribute to the documentation.

Based on the AnsibleFest talk from 2018. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ANSIBLE VERSION
2.8

### CODE REVIEW

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
+.. _community_documentation_contributions:
+
+*****************************************
+Contributing to the Ansible Documentation
+*****************************************
+
+Ansible has a lot of documentation and a small team of writers. Community support helps us keep up with new features, fixes, and changes.
+
+Improving the documentation is an easy way to make your first contribution to the Ansible project. You don't have to be a programmer, since our documentation is written in YAML (module documentation) or reStructured Text. If you're using Ansible, you know YAML - playbooks are written in YAML. And rST is mostly just text. You don't even need git experience, if you use the ``Edit on GitHub`` option.
```
Comment (sieben) [inline docs/docsite/rst/community/documentation_contributions.rst]: Is there a link that explain how to edit on github?

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+If you find a typo, a broken example, a missing topic, or any other error or omission on docs.ansible.com, let us know. Here are some ways to support Ansible documentation:
```
Comment (remyleone) [inline docs/docsite/rst/community/documentation_contributions.rst]: Could it be a link?

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+You can also contribute by reviewing open documentation issues and PRs. To add a helpful review, please:
+
+- Include a comment - "looks good to me" only helps if we know why.
+- For issues, reproduce the problem.
+- For PRs, test the change.
+
+Joining the documentation working group
+=======================================
+
+The Documentation Working Group is just getting started, please visit the community repo for more information.
```
Comment (remyleone) [inline docs/docsite/rst/community/documentation_contributions.rst]: Where is located this community repo?

**Comment by acozine** [comment]
Ready for review. The branch is published (temporarily) at http://docs.testing.ansible.com/ansible/latest/community/documentation_contributions.html#community-documentation-contributions and associated pages.

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 4]
```diff
@@ -0,0 +1,166 @@
+.. _community_documentation_contributions:
+
+*****************************************
+Contributing to the Ansible Documentation
+*****************************************
+
+Ansible has a lot of documentation and a small team of writers. Community support helps us keep up with new features, fixes, and changes.
+
+Improving the documentation is an easy way to make your first contribution to the Ansible project. You don't have to be a programmer, since our documentation is written in YAML (module documentation) or reStructured Text. If you're using Ansible, you know YAML - playbooks are written in YAML. And rST is mostly just text. You don't even need git experience, if you use the ``Edit on GitHub`` option.
```
Comment (felixfontein) [inline docs/docsite/rst/community/documentation_contributions.rst]: Shouldn't it be `YAML-playbooks` or `YAML playbooks`, and not `YAML - playbooks`?

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+   make htmlsingle rst=community/documentation_contributions.rst
+
+This method compiles all the links but provides minimal log output.
+
+.. note::
+
+    ``make htmlsingle`` adds ``rst/`` to the beginning of the path you provide in ``rst=``, so you can't type the filename with autocomplete. If you run ``make htmlsingle`` from the ``docs/docsite/rst/`` directory, you'll get ``make: *** No rule to make target `htmlsingle'.  Stop.`` If you run it from ``docs/docsite/`` and provide the full path, you'll get ``sphinx-build: error: cannot find files ['rst/rst/community/documentation_contributions.rst']``.
+
+2. Building an rST file with sphinx-build:
```
Comment (gundalow) [inline docs/docsite/rst/community/documentation_contributions.rst]: I wonder if you could his should be marked as advanced? I've never use it.

Maybe move to the end of the doc under and `advanced` section

File: docs/docsite/rst/community/documentation_contributions.rst (hunk 4 above)
Comment (gundalow) [reply docs/docsite/rst/community/documentation_contributions.rst]: `or reStructured Text`
`or reStructured Text (RST)`

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 6]
```diff
@@ -0,0 +1,166 @@
+.. _community_documentation_contributions:
```
Comment (gundalow) [inline docs/docsite/rst/community/documentation_contributions.rst:1]: Should Module DOCUMENTATION in dev_guide link to this?

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+.. code-block:: bash
+
+   make htmlsingle rst=community/documentation_contributions.rst
+
+This method compiles all the links but provides minimal log output.
+
+.. note::
+
+    ``make htmlsingle`` adds ``rst/`` to the beginning of the path you provide in ``rst=``, so you can't type the filename with autocomplete. If you run ``make htmlsingle`` from the ``docs/docsite/rst/`` directory, you'll get ``make: *** No rule to make target `htmlsingle'.  Stop.`` If you run it from ``docs/docsite/`` and provide the full path, you'll get ``sphinx-build: error: cannot find files ['rst/rst/community/documentation_contributions.rst']``.
```
Comment (gundalow) [inline docs/docsite/rst/community/documentation_contributions.rst]: Can we break this into steps?

**Review by gundalow — APPROVED** [review]

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+
+   make htmlsingle rst=path/to/your_file.rst
+
+For example:
+
+.. code-block:: bash
+
+   make htmlsingle rst=community/documentation_contributions.rst
+
+This process compiles all the links but provides minimal log output. If you're writing a new page or want more detailed log output, refer to the instructions on :ref:``
```
Comment (samccann) [inline docs/docsite/rst/community/documentation_contributions.rst]: missing what :ref: is pointing to here?

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+#. Fill out the PR template, including as much detail as appropriate for your change. You can change the title of your PR if you like (by default it's the same as your commit message). In the ``Issue Type`` section, delete all lines except the ``Docs Pull Request`` line.
+#. Submit your change by clicking on ``Create pull request`` button.
+#. Be patient while Ansibot, our automated script, adds labels, pings the docs maintainers, and kicks off a CI testing run.
+#. Keep an eye on your PR - the docs team may ask you for changes.
+
+Reviewing open PRs and issues
+=============================
+
+You can also contribute by reviewing open documentation issues and PRs. To add a helpful review, please:
+
```
Comment (samccann) [inline docs/docsite/rst/community/documentation_contributions.rst:38]: Is it worth adding a link to open doc PRs here? Or do we assume anyone looking to help with doc PRs will know how to find them?

File: docs/docsite/rst/community/documentation_contributions.rst (hunk 8 above)
Comment (acozine) [reply docs/docsite/rst/community/documentation_contributions.rst]: Zeut alors, I have missed one!

File: docs/docsite/rst/community/documentation_contributions.rst (hunk 9 above)
Comment (acozine) [reply docs/docsite/rst/community/documentation_contributions.rst:38]: Good idea.

Comment (acozine) [reply docs/docsite/rst/community/documentation_contributions.rst:38]: Added, thanks for suggesting that.

File: docs/docsite/rst/community/how_can_I_help.rst [hunk 10]
```diff
@@ -28,6 +28,11 @@ There are many forums online where Ansible users ask and answer questions. Reach
 
 You can find the official :ref:`Ansible communication channels <communication>`.
 
+Review, fix, and maintain the documentation
+===========================================
+
+Typos are everywhere, even in the Ansible documentation. We work hard to keep the documentation up-to-date, but you may also find out-dated examples. We offer easy ways to :ref:`report and/or fix documentation errors <community_documentation_contributions>`.
```
Comment (beeankha) [inline docs/docsite/rst/community/how_can_I_help.rst:34]: "out-dated" should be "outdated" instead.
```suggestion
Typos are everywhere, even in the Ansible documentation. We work hard to keep the documentation up-to-date, but you may also find outdated examples. We offer easy ways to :ref:`report and/or fix documentation errors <community_documentation_contributions>`.
```

**Review by samccann — APPROVED** [review]

**Pull request merged**